features/nfc/nfc/NFCController.h

```diff
@@ -16,6 +14,10 @@
  * limitations under the License.
  */
 
+/*
+ * @file NFCController.h
```

Comment:
Still not a doxygen commen, needs `/**`. Why do we need @file though?

Reply:
Rather than reverting the merged commit, I have just updated the current PR with @file specifier removed. This essentially brings back the original style as intended by you guys. Hope you can take over form here and resolve any doxy issues pointed out during docathlon.

Reply:
OK, so still will need to be fixed